projects/angular/src/forms/styles/_input-group.clarity.scss

```diff
@@ -56,11 +56,12 @@
 
     .clr-input-group-icon-action {
       @include mixins.clr-no-styles-button();
-      color: forms-variables.$clr-forms-focused-color;
+      color: forms-variables.$clr-forms-icon-color;
```

Comment:
I think that's not needed with the fill used under it.

Reply:
Corrected in commit 24cfa29